src/classes/UTIL_CustomSettingsFacade.cls

```diff
@@ -113,7 +113,7 @@ public without sharing class UTIL_CustomSettingsFacade {
         hs.Affl_ProgEnroll_Status_Map__c = 'Current';
         hs.Affiliation_Record_Type_Enforced__c = false;
         hs.Administrative_Account_Record_Type__c = UTIL_Describe.getAdminAccRecTypeID();
-
+        hs.Default_Contact_Language_Fluency__c = 'Fluent';
```

Comment:
The only thing I don't like about this is that for new customers/new installs, this will default to Fluent. For current customers, it's a blank value and they must go to the HEDA Settings UI and select a value.

Alternatively, I could set the default value of the custom setting field to "Fluent" right in the object metadata, but it doesn't seem like we've done that in the past so I wasn't sure on the best approach here...

Reply:
@robertbwatson We did something similar with Admin Account naming. I think our install script handled it, but upon install, if we found that the Admin RecordType wasn't specified, we set it to 'Administrative' then they could refer to it like they are doing on line 115 there.

Could we do something like this: https://github.com/SalesforceFoundation/HEDAP/blob/master/src/classes/STG_InstallScript.cls (line 262). This would set it for existing (because it would be empty since this is new). Just a thought.

Reply:
Actually, this is probably fine. I see that we are doing it for other options like 'Current' etc. That said, on install we could still probably add something like this to the Install Script so that it is consistent.

Reply:
@lifewithryan  Fantastic suggestion! I forgot about the install script, honestly. I'll look into this.

Reply:
@lifewithryan  Done - see STG_InstallScript & STG_InstallScript_TEST below. I'll update the WI to note a few extra tests we should consider with this approach.